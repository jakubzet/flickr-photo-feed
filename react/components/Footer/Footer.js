import React from "react";
import * as S from "./styled";

export const Footer = () => (
  <S.Footer>
    <S.Wrapper>
      &copy; 2019 <a href="http://pitahaya.pro">Jakub Å»ywiec</a>. All rights
      reserved. Respect your and other people&apos;s work.
    </S.Wrapper>
  </S.Footer>
);
