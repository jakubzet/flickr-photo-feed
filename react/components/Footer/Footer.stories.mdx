import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
import Footer from "./Footer";

<Meta
  title="Components|Footer"
  component={Footer}
  parameters={{ jest: ["Footer"] }}
/>

# Footer

Footer description placeholder.

<Preview>
  <Story name="default">
    <Footer />
  </Story>
</Preview>

<Props of={Footer} />