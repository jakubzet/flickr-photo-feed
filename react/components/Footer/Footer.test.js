import React from "react";
import { render } from "../../utils/reactTestingLibrary";
import { Footer } from "./Footer";

describe("Footer", () => {
  it("Renders", () => {
    const { container, asFragment } = render(<Footer />);

    expect(container).toBeDefined();
    expect(asFragment()).toMatchSnapshot();
  });
});
