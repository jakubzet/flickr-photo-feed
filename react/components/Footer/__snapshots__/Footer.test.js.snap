// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Renders 1`] = `
<DocumentFragment>
  <footer
    class="sc-bdVaJa cIDTaq"
  >
    <div
      class="sc-bwzfXH dZRTCS"
    >
      © 2019 
      <a
        href="http://pitahaya.pro"
      >
        Jakub Żywiec
      </a>
      . All rights reserved. Respect your and other people's work.
    </div>
  </footer>
</DocumentFragment>
`;
